doc/htconfig.md

```diff
@@ -1,14 +1,14 @@
 Config values that can only be set in .htconfig.php
 ===================================================
 
-There are some config values that haven't found their way into the administration page. This has several reasons. Maybe they are part of a 
-current development that isn't considered stable and will be added later in the administration page when it is considered safe. Or it triggers 
+There are some config values that haven't found their way into the administration page. This has several reasons. Maybe they are part of a
+current development that isn't considered stable and will be added later in the administration page when it is considered safe. Or it triggers
```

Comment:
I guess it should be one line per sentence? @tobiasd does know better about this. Of course then the original file would be wrong formatted as well :-)